lib/main.dart

```diff
@@ -0,0 +1,216 @@
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+import 'package:drift/drift.dart' as drift;
+import 'package:stopwatch_demo/stopwatch.dart';
+
+import 'database.dart' as Db;
```

Comment:
I have a warning (non blocker) on the `Db` name:

>The prefix 'Db' isn't a snake_case identifier.
Try changing the prefix to follow the snake_case style.

Reply:
Where are these warnings popping up? In VS Code? Just for future reference 👌

Reply:
Yes VSCode displays them in the `Problems` tab:
![image](https://user-images.githubusercontent.com/6057298/215735216-02675b04-d418-4124-bb49-70e8185bf8f8.png)